### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bermudezmt**
Clarify instructions on how to use the plugin in its current state.

### CODE REVIEW

**Comment by bermudezmt** [comment]
@vernig @kaiquelupo can you PTAL when you get a chance? Thanks!

File: README.md [hunk 1]
```diff
@@ -36,11 +36,20 @@ Via `homebrew`:
 $ brew tap twilio/brew && brew install twilio
 ```
 
+## Link the Twilio CLI Infra Plugin
+
+To get started, run `npm install` to install the plugin package dependencies. Afterwards, run the following command:
+
+```sh-session
+$ twilio plugins:link <path_to_your_local_repo>
```
Comment (vernig) [inline README.md]: @bermudezmt the `plugins:link` command is only needed to test local changes. When we merge all the fix to `main`, user will be able to use:
```
twilio plugins:install kaiquelupo/plugin-twilio-infra
```
or
```
twilio plugins:install twilio-infra-as-code/plugin-twilio-infra
```
 to install the plugin. 

The way the CLI works is that it uses `yarn` to install the plugin package. `yarn` is picking up the package either from npmjs or from github.

File: README.md [hunk 2]
```diff
@@ -36,11 +36,20 @@ Via `homebrew`:
...
+```
+
+You can now start using the `twilio infra` commands on your machine.
+
 ## Usage
 
 ```sh-session
-$ twilio plugins:install kaiquelupo/plugin-twilio-infra
-$ twilio --help infra
+$ twilio infra --help
```
Comment (vernig) [inline README.md:52]: Strangely enough both `twilio --help infra` and `twilio infra --help` works fine :-) Either way I'm fine with that

**Review by vernig — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (bermudezmt) [reply README.md]: You mean once this becomes an official npm package, right? Or are you actually able to run `twilio plugins:install` for this package now?

File: README.md (hunk 2 above)
Comment (bermudezmt) [reply README.md:52]: I prefer `twilio infra --help` as I think this is the convention Twilio is following? Also what I use to work with other CLI tools.

File: README.md (hunk 1 above)
Comment (vernig) [reply README.md]: You are able to do it already @bermudezmt without publishing the package on npm. The CLI uses `yarn` to fetch the plugin, and `yarn` can use github directly to fetch and install a new depenendecy. Just keep in mind that yarn is going to fetch the `main` branch of the repo.

Comment (bermudezmt) [reply README.md]: Thank you. I've updated.

**Comment by bermudezmt** [comment]
@kaiquelupo I've updated based on @vernig 's feedback. This is ready for merging. 😆

**Comment by vernig** [comment]
Thanks @bermudezmt 👍

**Pull request merged**